applications/dashboard/src/scripts/compatibilityStyles/buttonStylesCompat.ts

```diff
@@ -41,42 +41,64 @@ export const buttonCSS = () => {
     mixinButton(".BigButton:not(.Danger)", ButtonTypes.PRIMARY);
     mixinButton(".NewConversation.NewConversation", ButtonTypes.PRIMARY);
     mixinButton(".groupToolbar .Button.Primary", ButtonTypes.PRIMARY);
-    mixinButton(".BoxButtons .Button.Primary", ButtonTypes.PRIMARY);
+    mixinButton(".BoxButtons .ButtonGroup.Multi .Button.Primary", ButtonTypes.PRIMARY);
     mixinButton(".Section-Members .Group-RemoveMember", ButtonTypes.PRIMARY);
     mixinButton(".group-members-filter-box .Button.search", ButtonTypes.PRIMARY);
     mixinButton("#Form_Ban", ButtonTypes.PRIMARY);
     mixinButton(".Popup #UserBadgeForm button", ButtonTypes.PRIMARY);
     mixinButton(".Button.Handle", ButtonTypes.PRIMARY);
     mixinButton("div.Popup .Body .Button.Primary", ButtonTypes.PRIMARY);
     mixinButton(".ButtonGroup.Multi .Button.Handle", ButtonTypes.PRIMARY);
-    mixinButton(".ButtonGroup.Multi .Button.Handle .Sprite.SpDropdownHandle", ButtonTypes.PRIMARY);
+    // mixinButton(".ButtonGroup.Multi .Button.Handle .Sprite.SpDropdownHandle", ButtonTypes.PRIMARY);
+
+    const buttonBorderRadius = parseInt(formElementVars.border.radius.toString(), 10);
+    const borderOffset = globalVars.border.width * 2;
+    const handleSize = formElementVars.sizing.height - borderOffset;
+
+    if (buttonBorderRadius && buttonBorderRadius > 0) {
+        cssOut(`.ButtonGroup.Multi.NewDiscussion .Button.Handle .SpDropdownHandle::before`, {
+            marginTop: unit((formElementVars.sizing.height * 2) / 36), // center vertically
+            marginRight: unit(buttonBorderRadius * 0.035), // offset based on border radius. No radius will give no offset.
+            maxHeight: unit(handleSize),
+            height: unit(handleSize),
+            lineHeight: unit(handleSize),
+        });
+    }
 
     cssOut(`.ButtonGroup.Multi .Button.Handle .Sprite.SpDropdownHandle`, {
-        width: unit(formElementVars.sizing.height),
-        background: important("none"),
+        height: unit(handleSize),
+        maxHeight: unit(handleSize),
+        width: unit(handleSize),
+        maxWidth: unit(handleSize),
+        background: important("transparent"),
         backgroundColor: important("none"),
         ...borders({
-            ...globalVars.borderType.formElements.buttons,
             color: rgba(0, 0, 0, 0),
         }),
     });
 
+    cssOut(`.ButtonGroup.Multi.NewDiscussion .Button.Handle.Handle`, {
+        top: unit(0),
+        right: unit(formElementVars.border.width),
+        minWidth: importantUnit(handleSize),
```

Comment:
I can predict `veryImportantUnit` coming soon? (just a joke)
But naming convention is not clear. what is important and what is not?
Especially when we talk about some utils function like `unit`.
What function should I use next time I need to get unit?

Reply:
@alex-mtl This maps to CSS `!important`. https://css-tricks.com/when-using-important-is-the-right-choice/

Reply:
Got it.